org.freedesktop.Sdk.Extension.wxwidgets.metainfo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+  <id>org.freedesktop.Sdk.Extension.wxwidgets</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>wxWindows</project_license>
```

Comment:
This should do it
```suggestion
  <project_license>LGPL-2.1-or-later WITH WxWindows-exception-3.1</project_license>
```

(it's a valid SPDX identifier)

Reply:
Fixed in 10e425720ae72039fb551a46c0d369a6fe2e3dec